mmv1/third_party/terraform/services/composer/resource_composer_user_workloads_secret.go.erb

```diff
@@ -32,9 +32,9 @@ func ResourceComposerUserWorkloadsSecret() *schema.Resource {
 		},
 
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(120 * time.Minute),
-			Update: schema.DefaultTimeout(120 * time.Minute),
-			Delete: schema.DefaultTimeout(30 * time.Minute),
+			Create: schema.DefaultTimeout(time.Minute),
```

Comment:
what's the reason for changing the timeout time? Is this time change documented somewhere

Reply:
This is a follow up to https://github.com/GoogleCloudPlatform/magic-modules/pull/10466. Since these operations are very fast (taking less than a second on average), the previous timeout value was unnecessarily long.